.eslintrc.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
   },
   plugins: ['@typescript-eslint', 'import', 'jest', 'react'],
   extends: [
+    'eslint:recommended',
     'plugin:react/recommended',
     'plugin:react/jsx-runtime',
     'plugin:jest/recommended',
```

Comment:
We still have the `plugin:jest/recommended` extension and `jest` plugin in the global scope. Can we remove them as well?